### ISSUE TITLE
Why does useReducedMotionConfig exist separately from useReducedMotion?

### ISSUE THREAD

**Issue by quantizor** [framer/motion#1606]
Per title, I found this really confusing and it's not documented anywhere. At a minimum, please add documentation to the accessibility page stating the differences between the two hooks; ideally though, they should be merged and `useReducedMotion` should listen to the context value.

**Comment by pfernandom** [comment]
+1

**Comment by quantizor** [comment]
... thanks I guess
